.github/ISSUE_TEMPLATE/migration-issue.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Migration Issue
+about: Ask a question related to the go-genproto type migration
```

Comment:
Nit: Ask a question or report an issue related to...

Honestly don't really care, but also don't know if there are constraints on length of this field.

Reply:
done